zebra-rpc/src/methods/get_block_template_rpcs/types/get_mining_info.rs

```diff
@@ -0,0 +1,51 @@
+//! Response type for the `getmininginfo` RPC.
+
+use zebra_chain::parameters::Network;
+
+/// `chain` field of [`Response`] as defined in BIP70
+#[derive(Debug, PartialEq, Eq, serde::Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum ChainNetworkResponse {
+    /// Corresponds to Mainnet
+    Main,
+
+    /// Corresponds to Testnet
+    Test,
+}
+
+impl From<Network> for ChainNetworkResponse {
+    fn from(network: Network) -> Self {
+        match network {
+            Network::Mainnet => Self::Main,
+            Network::Testnet => Self::Test,
+        }
+    }
+}
+
+/// Response to a `getmininginfo` RPC request.
+#[derive(Debug, PartialEq, Eq, serde::Serialize)]
+pub struct Response {
+    /// The estimated network solution rate in Sol/s.
+    networksolps: u128,
+
+    /// The estimated network solution rate in Sol/s.
+    networkhashps: u128,
+
+    /// Current network name as defined in BIP70 (main, test, regtest)
+    chain: ChainNetworkResponse,
```

Comment:
Sorry I should have mentioned this before, this conversion is already implemented on Network:
https://github.com/ZcashFoundation/zebra/blob/6ade4354be61c4726e6beb4ce4251b479f619c2a/zebra-rpc/src/methods.rs#L379

Reply:
(I only remembered it was a field in another RPC after I made my first comment.)

Reply:
Oh. Got it. I just replaced that last commit with [`6e94c14`](https://github.com/ZcashFoundation/zebra/pull/5808/commits/6e94c14ddea5391e1a5a1c12fb22b5564979d7bf) which uses that instead.